Integrations/Integration-Wildfire.yml

```diff
@@ -148,19 +131,36 @@ script:
               return 'Something went wrong.\n Status code: ' + res.StatusCode + '\nBody: '+ JSON.stringify(res) + '.';
           break;
 
-      case 'WildFireReport':
+      case 'wildfire-report':
```

Comment:
Just verifying that this was the same command as before - we cannot change the commands or at least we need to support both.